src/app/demos/numeric/numeric-demo.component.html

```diff
@@ -95,6 +95,10 @@
         <td>15501 with truncation after 10000 and 1 digit</td>
         <td>{{15501 | skyNumeric:{digits: 1, truncateAfter: 10000} }}</td>
       </tr>
+      <tr>
+        <td>3712.4 as US dollar currency and a minimum of 4 decimal digits</td>
+        <td>{{ 3712.4 | skyNumeric: {minDigits: 4, format: 'currency', iso: 'USD', truncate: false} }}</td>
```

Comment:
Total nitpick here (sorry) - USD won't ever be portrayed with 4 decimals. Should we demonstrate the min with something more realistic? Maybe just switch this to 2? Or use one of the million examples in this list above?

Reply:
disregard.... not the public facing demo.